### ISSUE TITLE
Bepinex Unix Cannot identify executable type (got PE32 executable (GUI) Intel 80386, for MS Windows)!

### ISSUE THREAD

**Issue by Duoun** [BepInEx/BepInEx#275]
**Describe the issue**
run_bepinex.sh Isn't working

**To Reproduce**
Take the bepinex Unix Zip and put it into the game's folder. Then make the run_bepinex.sh script executable. edit the script with notepad++ to say "RainWorld.exe" then I run it in terminal.

**Expected behavior**
Responds with `Cannot identify executable type (got PE32 executable (GUI) Intel 80386, for MS Windows)!`

**Screenshots and logs**
![help](https://user-images.githubusercontent.com/87386719/125501424-2d98f65b-f2ea-4f3a-bf0e-5efe8bed432a.png)

**Desktop (please complete the following information):**
 - Rain World 1.5 Version (Update)
 - BepInEx version 5.4.11

**Additional context**
There exists a Bepinex Zip file that was made for Rain World modding but as it turns out you need to do something special if you on Linux. I was sent to this Link by the Rain World Modding Support Discord [https://docs.bepinex.dev/articles/user_guide/installation/index.html?tabs=tabid-nix](url) Also this is my first time sending an issue on GitHub so sorry if it doesn't look good.

**Comment by ghorsington** [comment]
Greetings!

Thank you for the information. The reported message is correct and implies that you are trying to use BepInEx targeted for native *nix Unity games on a Unity game compiled for Windows and is being run through Proton/Wine.

In that case, you want to use normal BepInEx builds for Windows and instead configure Proton or Wine to detect and make use of BepInEx properly. A general guide for it on BepInEx Docs is [Steam interop with Proton/Wine](https://docs.bepinex.dev/articles/advanced/steam_interop.html#protonwine).

**Comment by Duoun** [comment]
I have winhttp added but the game still doesn't even load

**Comment by FoxinTale** [comment]
Figured out the solution to this by messing around, as I was initially having this problem myself. Assuming the game is native to Windows and is being run through Proton, with only a ".exe", install the Windows version of BepInEx, then follow the very last part of the guide [proxying winhttp.dll through wine](https://docs.bepinex.dev/articles/advanced/steam_interop.html#open-winecfg-for-the-target-game),  You'll want to install protontricks via your package manager, and if it says it doesn't exist, search it. Arch said via pacman it didn't exist, but trizen found it. Should then be able to launch the game as normal. I tried this for Subnautica, but it should work for most other games.
